src/app/components/server/Icons.tsx

```diff
@@ -381,7 +381,7 @@ export const ClockIcon = (props: SVGProps<SVGSVGElement> & { alt: string }) => {
       <path
         fillRule="evenodd"
         clipRule="evenodd"
-        d="M12 4C10.4178 4 8.87103 4.49515 7.55544 5.42283C6.23985 6.35051 5.21447 7.66906 4.60897 9.21174C4.00347 10.7544 3.84504 12.4519 4.15372 14.0896C4.4624 15.7273 5.22433 17.2317 6.34315 18.4124C7.46197 19.5931 8.88743 20.3972 10.4393 20.7229C11.9911 21.0487 13.5997 20.8815 15.0615 20.2425C16.5233 19.6035 17.7727 18.5214 18.6518 17.133C19.5308 15.7446 20 14.1124 20 12.4426C19.9976 10.2042 19.154 8.05829 17.6542 6.47555C16.1545 4.8928 14.121 4.00251 12 4V4ZM12 18.7745C10.8133 18.7745 9.65328 18.4031 8.66658 17.7074C7.67989 17.0116 6.91085 16.0227 6.45673 14.8657C6.0026 13.7087 5.88378 12.4355 6.11529 11.2073C6.3468 9.979 6.91825 8.85076 7.75736 7.96522C8.59648 7.07968 9.66558 6.47663 10.8295 6.23231C11.9933 5.98799 13.1997 6.11338 14.2961 6.59263C15.3925 7.07188 16.3295 7.88346 16.9888 8.92474C17.6481 9.96602 18 11.1902 18 12.4426C17.9982 14.1213 17.3654 15.7307 16.2406 16.9178C15.1158 18.1048 13.5907 18.7725 12 18.7745ZM15.5 12.4426H12V8.74894C12 8.609 11.9473 8.47479 11.8536 8.37583C11.7598 8.27688 11.6326 8.22128 11.5 8.22128C11.3674 8.22128 11.2402 8.27688 11.1465 8.37583C11.0527 8.47479 11 8.609 11 8.74894V12.9702C11 13.1102 11.0527 13.2444 11.1465 13.3433C11.2402 13.4423 11.3674 13.4979 11.5 13.4979H15.5C15.6326 13.4979 15.7598 13.4423 15.8536 13.3433C15.9473 13.2444 16 13.1102 16 12.9702C16 12.8303 15.9473 12.6961 15.8536 12.5971C15.7598 12.4982 15.6326 12.4426 15.5 12.4426Z"
+        d="M12 0C9.62663 0 7.30655 0.703788 5.33316 2.02236C3.35977 3.34094 1.8217 5.21508 0.913451 7.4078C0.00519937 9.60051 -0.232441 12.0133 0.230582 14.3411C0.693605 16.6689 1.83649 18.807 3.51472 20.4853C5.19295 22.1635 7.33115 23.3064 9.65892 23.7694C11.9867 24.2324 14.3995 23.9948 16.5922 23.0865C18.7849 22.1783 20.6591 20.6402 21.9776 18.6668C23.2962 16.6934 24 14.3734 24 12C23.9964 8.8185 22.731 5.76833 20.4813 3.51866C18.2317 1.269 15.1815 0.00357316 12 0ZM12 21C10.22 21 8.47992 20.4722 6.99987 19.4832C5.51983 18.4943 4.36628 17.0887 3.68509 15.4441C3.0039 13.7996 2.82567 11.99 3.17294 10.2442C3.5202 8.49836 4.37737 6.89471 5.63604 5.63604C6.89472 4.37736 8.49836 3.5202 10.2442 3.17293C11.99 2.82566 13.7996 3.00389 15.4442 3.68508C17.0887 4.36627 18.4943 5.51982 19.4832 6.99987C20.4722 8.47991 21 10.22 21 12C20.9972 14.3861 20.0481 16.6737 18.3609 18.3609C16.6737 20.0481 14.3861 20.9972 12 21ZM17.25 12H12V6.75C12 6.55109 11.921 6.36032 11.7803 6.21967C11.6397 6.07902 11.4489 6 11.25 6C11.0511 6 10.8603 6.07902 10.7197 6.21967C10.579 6.36032 10.5 6.55109 10.5 6.75V12.75C10.5 12.9489 10.579 13.1397 10.7197 13.2803C10.8603 13.421 11.0511 13.5 11.25 13.5H17.25C17.4489 13.5 17.6397 13.421 17.7803 13.2803C17.921 13.1397 18 12.9489 18 12.75C18 12.5511 17.921 12.3603 17.7803 12.2197C17.6397 12.079 17.4489 12 17.25 12Z"
```

Comment:
Hmm, I did not expect icons to need changing - especially since SVGs are supposed to be scalable, so you should be able to set the desired size at the point of use too.

Did you check where else this icon is used, and if it still looks the same there as before this PR? If it does, this change should be fine, but if you haven't checked, that's probably worth doing :)